packages/backend/database/src/drizzle/schema.ts

```diff
@@ -0,0 +1,454 @@
+import {
+  pgTable,
+  varchar,
+  integer,
+  boolean,
+  timestamp,
+  text,
+  uniqueIndex,
+  primaryKey,
+  index,
+} from "drizzle-orm/pg-core";
+
+export const user = pgTable(
+  "user",
+  {
+    id: varchar("id", { length: 255 }).primaryKey(),
+    email: varchar("email", { length: 30 }).notNull(),

```

Comment:
email should be unique.     email: varchar("email", { length: 50 }).notNull().unique(),